<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1108</width>
    <height>853</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="glView" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>19</x>
      <y>19</y>
      <width>760</width>
      <height>760</height>
     </rect>
    </property>
   </widget>
   <widget class="QSlider" name="sliderScaling">
    <property name="geometry">
     <rect>
      <x>860</x>
      <y>40</y>
      <width>160</width>
      <height>25</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>30</number>
    </property>
    <property name="sliderPosition">
     <number>15</number>
    </property>
    <property name="tracking">
     <bool>true</bool>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSlider" name="sliderMoveX">
    <property name="geometry">
     <rect>
      <x>860</x>
      <y>280</y>
      <width>160</width>
      <height>25</height>
     </rect>
    </property>
    <property name="minimum">
     <number>-15</number>
    </property>
    <property name="maximum">
     <number>15</number>
    </property>
    <property name="sliderPosition">
     <number>0</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSlider" name="sliderMoveY">
    <property name="geometry">
     <rect>
      <x>860</x>
      <y>330</y>
      <width>160</width>
      <height>25</height>
     </rect>
    </property>
    <property name="minimum">
     <number>-15</number>
    </property>
    <property name="maximum">
     <number>15</number>
    </property>
    <property name="sliderPosition">
     <number>0</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSlider" name="sliderMoveZ">
    <property name="geometry">
     <rect>
      <x>860</x>
      <y>380</y>
      <width>160</width>
      <height>25</height>
     </rect>
    </property>
    <property name="minimum">
     <number>-15</number>
    </property>
    <property name="maximum">
     <number>15</number>
    </property>
    <property name="sliderPosition">
     <number>0</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSlider" name="sliderRotateZ">
    <property name="geometry">
     <rect>
      <x>860</x>
      <y>210</y>
      <width>160</width>
      <height>25</height>
     </rect>
    </property>
    <property name="maximum">
     <number>360</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSlider" name="sliderRotateX">
    <property name="geometry">
     <rect>
      <x>860</x>
      <y>110</y>
      <width>160</width>
      <height>25</height>
     </rect>
    </property>
    <property name="maximum">
     <number>360</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSlider" name="sliderRotateY">
    <property name="geometry">
     <rect>
      <x>860</x>
      <y>160</y>
      <width>160</width>
      <height>25</height>
     </rect>
    </property>
    <property name="maximum">
     <number>360</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="selectFile">
    <property name="geometry">
     <rect>
      <x>860</x>
      <y>450</y>
      <width>161</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Select file</string>
    </property>
   </widget>
   <widget class="QPushButton" name="scalingDown">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>36</y>
      <width>21</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QPushButton" name="scalingUp">
    <property name="geometry">
     <rect>
      <x>830</x>
      <y>36</y>
      <width>21</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>+</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="inputValueScale">
    <property name="geometry">
     <rect>
      <x>1030</x>
      <y>40</y>
      <width>62</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimum">
     <double>0.100000000000000</double>
    </property>
    <property name="maximum">
     <double>3.000000000000000</double>
    </property>
    <property name="singleStep">
     <double>0.100000000000000</double>
    </property>
    <property name="value">
     <double>1.500000000000000</double>
    </property>
   </widget>
   <widget class="QPushButton" name="rotateXDown">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>105</y>
      <width>21</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QPushButton" name="rotateXUp">
    <property name="geometry">
     <rect>
      <x>830</x>
      <y>105</y>
      <width>21</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>+</string>
    </property>
   </widget>
   <widget class="QPushButton" name="rotateYUp">
    <property name="geometry">
     <rect>
      <x>830</x>
      <y>155</y>
      <width>21</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>+</string>
    </property>
   </widget>
   <widget class="QPushButton" name="rotateYDown">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>155</y>
      <width>21</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QPushButton" name="rotateZUp">
    <property name="geometry">
     <rect>
      <x>830</x>
      <y>205</y>
      <width>21</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>+</string>
    </property>
   </widget>
   <widget class="QPushButton" name="rotateZDown">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>205</y>
      <width>21</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QPushButton" name="moveXDown">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>275</y>
      <width>21</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QPushButton" name="moveXUp">
    <property name="geometry">
     <rect>
      <x>830</x>
      <y>275</y>
      <width>21</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>+</string>
    </property>
   </widget>
   <widget class="QPushButton" name="moveYDown">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>325</y>
      <width>21</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QPushButton" name="moveYUp">
    <property name="geometry">
     <rect>
      <x>830</x>
      <y>325</y>
      <width>21</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>+</string>
    </property>
   </widget>
   <widget class="QPushButton" name="moveZDown">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>375</y>
      <width>21</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QPushButton" name="moveZUp">
    <property name="geometry">
     <rect>
      <x>830</x>
      <y>375</y>
      <width>21</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>+</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="inputValueRotateX">
    <property name="geometry">
     <rect>
      <x>1030</x>
      <y>110</y>
      <width>62</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimum">
     <double>0.000000000000000</double>
    </property>
    <property name="maximum">
     <double>360.000000000000000</double>
    </property>
    <property name="singleStep">
     <double>1.000000000000000</double>
    </property>
    <property name="value">
     <double>0.000000000000000</double>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="inputValueRotateY">
    <property name="geometry">
     <rect>
      <x>1030</x>
      <y>160</y>
      <width>62</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimum">
     <double>0.000000000000000</double>
    </property>
    <property name="maximum">
     <double>360.000000000000000</double>
    </property>
    <property name="singleStep">
     <double>1.000000000000000</double>
    </property>
    <property name="value">
     <double>0.000000000000000</double>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="inputValueRotateZ">
    <property name="geometry">
     <rect>
      <x>1030</x>
      <y>210</y>
      <width>62</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimum">
     <double>0.000000000000000</double>
    </property>
    <property name="maximum">
     <double>360.000000000000000</double>
    </property>
    <property name="singleStep">
     <double>1.000000000000000</double>
    </property>
    <property name="value">
     <double>0.000000000000000</double>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="inputValueMoveX">
    <property name="geometry">
     <rect>
      <x>1030</x>
      <y>280</y>
      <width>62</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimum">
     <double>-15.000000000000000</double>
    </property>
    <property name="maximum">
     <double>15.000000000000000</double>
    </property>
    <property name="singleStep">
     <double>1.000000000000000</double>
    </property>
    <property name="value">
     <double>0.000000000000000</double>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="inputValueMoveY">
    <property name="geometry">
     <rect>
      <x>1030</x>
      <y>330</y>
      <width>62</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimum">
     <double>-15.000000000000000</double>
    </property>
    <property name="maximum">
     <double>15.000000000000000</double>
    </property>
    <property name="singleStep">
     <double>1.000000000000000</double>
    </property>
    <property name="value">
     <double>0.000000000000000</double>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="inputValueMoveZ">
    <property name="geometry">
     <rect>
      <x>1030</x>
      <y>380</y>
      <width>62</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimum">
     <double>-15.000000000000000</double>
    </property>
    <property name="maximum">
     <double>15.000000000000000</double>
    </property>
    <property name="singleStep">
     <double>1.000000000000000</double>
    </property>
    <property name="value">
     <double>0.000000000000000</double>
    </property>
   </widget>
   <widget class="QLabel" name="numbOfVertex">
    <property name="geometry">
     <rect>
      <x>980</x>
      <y>510</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="numbOfSurface">
    <property name="geometry">
     <rect>
      <x>980</x>
      <y>530</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="labelNameFile">
    <property name="geometry">
     <rect>
      <x>980</x>
      <y>550</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>860</x>
      <y>550</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>File name:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>860</x>
      <y>510</y>
      <width>58</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Vertexes:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>860</x>
      <y>530</y>
      <width>58</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Edges:</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1108</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>glView</class>
   <extends>QWidget</extends>
   <header>glview.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
